src/make_tools.ml

```diff
@@ -109,6 +109,8 @@ let () = if build_macGUI then outp "macuimaybe: macui"
 (* Generate backtrace information for exceptions *)
 let () = "CAMLFLAGS" <-+= "-g $(INCLFLAGS)"
 
+let () = "CAMLCFLAGS" <-+= "-ccopt -D_FILE_OFFSET_BITS=64"
```

Comment:
Is `_FILE_OFFSET_BITS` defined by POSIX, or is this a glibc thing?   I've never been clear on "large file support" because NetBSD simply changed `off_t` etc. to be 64-bit types long ago, regardless of the CPU type, so there's never been any need for or discussion of "large file support"; large files have simply worked with the normal syscalls.  (I have been boggled at how other systems have a parallel set of calls, which seems awkward, but that's off topic.)

I couldn't find this in POSIX, which makes me think it should be defined only where needed.   And, I don't understand why it's new with this change, as unison should already handle files over 2 GB.

Reply:
This discussion happened over at #876 (although the change always originated from this PR).

> And, I don't understand why it's new with this change, as unison should already handle files over 2 GB.

That is correct, OCaml Unix library has had this defined since 2002. However, this is the first time we have C code touching `off_t` directly. It is for the benefit of this C code.

Reply:
That would be a good comment to explain it, then, that it aligns with what ocaml is already doing